mmv1/third_party/terraform/website/docs/r/apigee_flowhook.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
+subcategory: "Apigee"
+description: |-
+  Represents a sharedflow attacehment to a flowhook point.
+---
+
+# google\_apigee\_flowhook
+
+Represents a sharedflow attacehment to a flowhook point.
+
+
+To get more information about Flowhook, see:
+
+* [API documentation](https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.environments.flowhooks#FlowHook)
+* How-to Guides
+    * [organizations.environments.flowhooks](https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.environments.flowhooks#FlowHook)
+
+## Argument Reference
+
+The following arguments are supported:
+
+
+* `org_id` -
+  (Required)
+  The Apigee Organization associated with the environment
+
+* `environment` -
+  (Required)
+  The resource ID of the environment.
+
+* `flow_hook_point` -
+  (Required)
+  Where in the API call flow the flow hook is invoked. Must be one of PreProxyFlowHook, PostProxyFlowHook, PreTargetFlowHook, or PostTargetFlowHook.
+
+* `description` -
+  (Required)
+  Description of the flow hook.
+
+* `sharedflow` -
+  (Required)
```

Comment:
I think this is probably optional as the [documentation says "empty if there is none attached"](https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.environments.flowhooks#FlowHook)

Reply:
> I implemented the resource to represent the "binding/attachment" between a flowhook and a sharedflow
Is this why this is required then?